docs/index.rst

```diff
@@ -1,49 +1,168 @@
+####################################
+
+Welcome new and returning Open edX developers!
+
+This is the central repository for documentation that will help you work with Open edX code.
+
+
+New to Open edX development? Start here
+=======================================
+
+This quick overview is intended for developers new to the Open edX platform, junior and senior alike.
+
+After you've reviewed these topics you'll be ready for more detail on our architecture, or you can prepare for active development with the topics in "Information for all Open edX developers."
+
+* :doc:`new_to_edx/mission`
+* :doc:`Our code of conduct <universal_stub>`
+* `A 15-minute introduction to Open edX systems <https://edx.readthedocs.io/projects/edx-developer-guide/en/latest/architecture.html>`_
+* :doc:`More about each system <new_to_edx/what_our_systems_do>`
+* :doc:`new_to_edx/data_model`
+* :doc:`new_to_edx/messaging_model`
+* :doc:`What you as a developer can do with the Open edX platform <new_to_edx/what_you_can_do>`
 
-This the landing page for documentation relevant to developers of the Open edX platform.  For other documentation
-resources targeted for edX learners, educators, researchers, and Open edX operators, see `docs.edx.org <docs.edx.org>`_.
 
 .. toctree::
-    :hidden:
+   :hidden:
+   :maxdepth: 4
+   :caption: New to Open edX development?
+
+   new_to_edx/mission
+   Our code of conduct <universal_stub>
+   A 15-minute introduction to Open edX systems <https://edx.readthedocs.io/projects/edx-developer-guide/en/latest/architecture.html>
+   More about each system <new_to_edx/what_our_systems_do>
+   new_to_edx/data_model
+   new_to_edx/messaging_model
+   What you as a developer can do with the Open edX platform <new_to_edx/what_you_can_do>
+
+
+Information for all Open edX developers
+=======================================
 
-    Open edX Proposals <https://open-edx-proposals.readthedocs.io/en/latest/>
-    developers_guide/index
-    named_releases
-    Open edX Extensions and APIs <https://open.edx.org/extending-edx>
-    (Legacy) Open edX Developer's Guide <https://edx.readthedocs.io/projects/edx-developer-guide/en/latest/>
+You will likely use some or all of the information in this section as you progress
+in your Open edX development work. Take a look through the topics here before you begin
+writing code.
 
 
-Architecture and Best Practices
--------------------------------
+.. From the outline: these topics appear to be of common interest. Are they? i18n, XSS, MFEs (Micro-frontends), IDAs (Independently Deployed Applications, a.k.a. Microservices), Django Configuration/Toggles, Named Releases]
 
 .. list-table::
+   :widths: auto
 
-   * - Open edX Technology Radar (coming...)
-     - A bird's eye view of the decisions, tools and technologies that have been adopted or rejected for Open edX
-       development.
+   * - :doc:`all_development/roadmap/index`
+     - Getting your device set up for development, testing, and deployment.
```

Comment:
1. Why isn't setting up your development environment for part of "New to Open edX development"?
2. I like "computer" or "machine" better than "device", but maybe that's just me.  Thoughts?

Reply:
1. I want "new to Ed X" to be a friendly, "have a coffee and a bit of info" section. Let's leave the wrangling of Devstack to the next section. 
2. I prefer "device" because it doesn't exclude virtual / cloud deployments. "Computer" sounds to me like a physical entity, and "machine" has connotations I'd rather avoid. If we need to we can take a little informal poll of our reviewers here; I'm willing to go with popular opinion.

Reply:
See commit 9a26b06

Reply:
I COPIED the setup info to the New to edX section so it will remain in the "all devs" section as well. And "device" has been removed everywhere I found it.

Reply:
Perhaps we can sidestep "device" entirely with "Getting set up for development, testing, and deployment."  Or, "Getting yourself set up ..." ?

Reply:
> Perhaps we can sidestep "device" entirely with "Getting set up for development, testing, and deployment." Or, "Getting yourself set up ..." ?

@nedbat the word "device" has been removed entirely